### PR TITLE
Docs: Combining Update, Create and Delete on one page

### PR DESCRIPTION
**Pull request by mdmunir**
Documentation for tabular input. Sorry i am not good in english :grin:

### CODE REVIEW

**Comment by mdmunir** [comment]
https://github.com/mdmsoft/yii2-widgets

**Comment by cebe** [comment]
still would be good to have at least some words around the code :)
Does not need to be perfect english, it's important that there is description of what is going on and how the code works. You can write a draft, we can polish the english afterwards.

**Comment by mdmunir** [comment]
I hope i am doing right :grinning:

**Comment by cebe** [comment]
its a good start, thank you. I will review it later, if you want you can add even more explaination.

File: docs/guide/input-tabular-input.md [hunk 1]
```diff
@@ -105,8 +105,112 @@ In the view you can use javascript to add new input lines dynamically.
...
 ### Combining Update, Create and Delete on one page
 
-> Note: This section is under development.
->
-> It has no content yet.
+To combine create, update, delete on one page, We can do the following step.
+* Create your controller action. 
 
-TBD
+You can use other name like `actionCreateOrUpdate()` or anyting else.
```
Comment (Faryshta) [inline docs/guide/input-tabular-input.md]: shouldn't it be naked `actionSave()` ?

File: docs/guide/input-tabular-input.md [hunk 2]
```diff
@@ -105,8 +105,112 @@ In the view you can use javascript to add new input lines dynamically.
...
+First, we load original data. In posting request, we compare original data and posting data. For original data that
+posted again, update it. For original data that not posted, delete it. For new data, created it.
+```php
+public function actionCreate()
+{
+    $settings = $origins = Setting::findAll();
```
Comment (SilverFire) [inline docs/guide/input-tabular-input.md]: Bad example. `User::findAll()` over 100k table is not the best option

**Review by SilverFire — CHANGES_REQUESTED** [review]

**Comment by samdark** [comment]
Closing since can't be merged as is and wasn't finished.

**Pull request closed without merging**